cypress/integration/pages/onDemandAudio/tests.js

```diff
@@ -44,79 +44,87 @@ export default ({ service, pageType, variant, isAmp }) => {
       before(() => {
         cy.getToggles(service);
       });
-      describe('Recent Episodes component', () => {
-        it('should be displayed if the toggle is on, and shows the expected number of items', function test() {
-          let toggleName;
-          if (Cypress.env('currentPath').includes('podcasts')) {
-            toggleName = 'recentPodcastEpisodes';
-          } else {
-            toggleName = 'recentAudioEpisodes';
-          }
-          cy.fixture(`toggles/${service}.json`).then(toggles => {
-            const recentEpisodesEnabled = path(
-              [toggleName, 'enabled'],
-              toggles,
-            );
-            cy.log(
-              `Recent Episodes component enabled? ${recentEpisodesEnabled}`,
-            );
-            // There cannot be more episodes shown than the max allowed
-            if (recentEpisodesEnabled) {
-              const recentEpisodesMaxNumber = path(
-                [toggleName, 'value'],
+      describe(
+        'Recent Episodes component',
+        {
+          retries: 3,
+        },
+        () => {
+          it('should be displayed if the toggle is on, and shows the expected number of items', function test() {
+            // Reload for retry if data didn't update on page
+            cy.reload();
```

Comment:
This reload() will run regardless of whether it's a retried test or not right? I wonder if this might solve the problem, regardless of retries.

1. Do we have a benchmark for how long this test suite takes to run without the retries (so that we can compare that with how long they take to run after the retries are added)? 
2. Do we have any way of tracking if this test **is** retrying once deployed? I.e. if we stop seeing the test flakes, can we know if it's the retries that have fixed it, or calling reload()?